WindowsServerDocs/remote/remote-access/directaccess/single-server-wizard/da-basic-plan-s1-infrastructure.md

```diff
@@ -110,6 +110,17 @@ In a DirectAccess deployment, DNS is required for the following:
 
     2.  If the corporate network is IPv6-based, the default address is the IPv6 address of DNS servers in the corporate network.
 
+> [!NOTE]
+> Starting with the Windows 10 May 2020 Update, a client no longer registers its IP addresses on DNS servers configured in a Name Resolution Policy (NRPT).
+> If the DNS registration is needed, for e.g. Manage Out, it can be explicitely enabled with this registry key on the client:
+> 
+> Path: ```HKLM\System\CurrentControlSet\Services\Dnscache\Parameters```   
+> Type: ``DWORD``   
+> Value name: ``DisableNRPTForAdapterRegistration``   
+> Values:   
+> ``1`` - DNS Registration disabled (default since Windows 10 May 2020 Update)   
+> ``0`` - DNS Registration enabled
```

Comment:
According to MarkDown best practice rules, only 1 single pair of code encapsulation backticks is needed for a horizontal (non-wrapped) string. 3 backticks can be used in a code block fence (multi-line). I have never seen a double pair used before, do you have a rationale for this uncommon usage? Let me suggest the common usage based on GitHub Flavored MarkDown (GFM):
```suggestion
> Path: `HKLM\System\CurrentControlSet\Services\Dnscache\Parameters`   
> Type: `DWORD`
> Value name: `DisableNRPTForAdapterRegistration`
> Values:
> `1` - DNS Registration disabled (default since Windows 10 May 2020 Update)
> `0` - DNS Registration enabled
```
Also removed some redundant end-of-line whitespace (blanks).